### PR TITLE
Change owner to pptruser

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

**Comment by jecfish** [comment]
@steren PTAL

File: user-journeys/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM ghcr.io/puppeteer/puppeteer:16.1.0
-COPY package*.json ./
+COPY --chown=pptruser:pptruser package*.json ./
```
Comment (jecfish) [inline user-journeys/Dockerfile:2]:
```suggestion
COPY package*.json ./
```

File: user-journeys/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM ghcr.io/puppeteer/puppeteer:16.1.0
```
[the remark this answers is not in the document]
Comment (jecfish) [reply user-journeys/Dockerfile:1]: Tried it, it's not working. 

The current USER is already `pptruser` (inherit from pptr docker, see [this](https://github.com/puppeteer/puppeteer/blob/main/docker/Dockerfile)), that's why the permission is failed.

However, the `package*.json` file doesn't need chown. So I removed that. In fact, we only need to chmod the `recordings` and `screenshot` (if we take screenshot) folders .

**Review by steren — APPROVED** [review]

**Comment by jecfish** [comment]
Thanks! Please help to merge as I am not authorized.

**Pull request merged**